modules/mobile/app/controllers/mobile/v0/maintenance_windows_controller.rb

```diff
@@ -20,10 +19,10 @@ class MaintenanceWindowsController < ApplicationController
         %i[vbms evss],
         %i[vbms lighthouse],
         %i[lighthouse claims],
-        %i[lighthouse direct_deposit_benefits],
-        %i[lighthouse disability_rating],
-        %i[lighthouse letters_and_documents],
-        %i[lighthouse immunizations],
```

Comment:
My understanding is that lighthouse should take down immunizations. Did you get info from someone to the contrary?

Reply:
Ah right, forgot to push that change. I originally removed it because I was going to skip lighthouse altogether because currently pager duty is manually populated. I meant to add it back though

Reply:
Fixing that, if it's wrong, is part of the larger ticket. This was just to decouple evss and lighthouse.

Reply:
Ya there will be another ticket to breakdown `lighthouse` into a maintenance window for each endpoint we use. That process is all still being talked about but we'll be able to make this more specific